api/admin/controller.py

```diff
@@ -2410,26 +2425,93 @@ def _sitewide_settings_controller(self, configuration_object):
         setting.value = value
         return Response(unicode(setting.key), 200)
 
-    def collections(self):
-        provider_apis = [OPDSImporter,
-                         OPDSForDistributorsAPI,
-                         OverdriveAPI,
-                         OdiloAPI,
-                         BibliothecaAPI,
-                         Axis360API,
-                         OneClickAPI,
-                         EnkiAPI,
-                         ODLWithConsolidatedCopiesAPI,
-                         SharedODLAPI,
-                         FeedbooksOPDSImporter,
-                        ]
+    def _get_protocols(self, provider_apis):
```

Comment:
I would rename this `_get_collection_protocols` to avoid confusion with `_get_integration_protocols`.